### PR TITLE
#824 improve watch slideshare link in talks

### PR DESCRIPTION
**Pull request by tkharuk**
Resolves [github issue](https://github.com/pivorakmeetup/pivorak-web-app/issues/x)

### Description

Describe the changes and motivations for the pull request, unless obvious from the title.

### How to test instructions

Describe how to test your changes if specs are not covering completely.

### CODE REVIEW

File: app/assets/stylesheets/app/base/_link.scss [hunk 1]
```diff
@@ -32,28 +32,4 @@ $namespace: ".pk-link";
       border-color: transparent;
     }
   }
-}
-
-#{$namespace}--btn-slides {
```
Comment (skrix) [inline app/assets/stylesheets/app/base/_link.scss:37]: Have some doubts that it was used somewhere else, not only on one button

File: app/helpers/talks_helper.rb [hunk 2]
```diff
@@ -17,7 +17,7 @@ def talk_video_url(talk, talk_content, options = {})
 
   def talk_slides_url(talk, options = {})
     options[:target] = '_blank'
-    link_to 'check slides', talk.slides_url, options if talk.slides_url.present?
+    link_to 'Watch slideshare', talk.slides_url, options if talk.slides_url.present?
```
Comment (skrix) [inline app/helpers/talks_helper.rb]: The text should be "Watch slides" )

**Review by skrix — CHANGES_REQUESTED** [review]
Need some improvement and more checks for not breaking anything

**Comment by tkharuk** [comment]
Buttons actually would like slightly different.
Unsure how https://github.com/pivorakmeetup/pivorak-web-app/pull/829 changes "Edit" button styles

image

**Comment by skrix** [comment]
Thanks for your efforts, but I should close this pr for now in cause of probable rewriting of this part.

**Pull request closed without merging**